src/dialog/unix/SDL_zenitydialog.c

```diff
@@ -18,53 +18,32 @@
      misrepresented as being the original software.
   3. This notice may not be removed or altered from any source distribution.
 */
-#include "SDL_internal.h"
 #include "../SDL_dialog_utils.h"
+#include "SDL_internal.h"
```

Comment:
This header needs to be included first, otherwise precompiled headers will break.